osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -0,0 +1,203 @@
+                        {
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Direction = FillDirection.Vertical,
+                        },
+                        new Container
+                        {
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Children = new Drawable[]
+                            {
+                                new Box
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                    Colour = OsuColour.Gray(0.2f)
+                                },
+                                new FillFlowContainer
+                                {
+                                    RelativeSizeAxes = Axes.X,
+                                    AutoSizeAxes = Axes.Y,
+                                    Direction = FillDirection.Vertical,
+                                    Children = new Drawable[]
+                                    {
+                                        deletedChildsPlaceholder = new DeletedChildsPlaceholder
+                                        {
+                                            ShowDeleted = { BindTarget = ShowDeleted }
+                                        },
+                                        new Container
+                                        {
+                                            AutoSizeAxes = Axes.Y,
+                                            RelativeSizeAxes = Axes.X,
+                                            Child = moreButton = new CommentsShowMoreButton
+                                            {
+                                                Anchor = Anchor.Centre,
+                                                Origin = Anchor.Centre,
+                                                Margin = new MarginPadding(5),
+                                                Action = () => getComments(false),
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            });
+        }
+
+        protected override void LoadComplete()
+        {
+            Sort.BindValueChanged(onSortChanged, true);
+            base.LoadComplete();
+        }
+
+        private void onSortChanged(ValueChangedEvent<CommentsSortCriteria> sort) => getComments();
+
+        private void getComments(bool initial = true)
+        {
+            if (initial)
+            {
+                currentPage = 1;
+                loadedTopLevelComments = 0;
+                deletedChildsPlaceholder.DeletedCount.Value = 0;
+                moreButton.IsLoading = true;
+                content.Clear();
+            }
```

Comment:
Does this need to be in here behind a flag, instead of somewhere where setup/loading is normally done? (be it ctor, or `Load()`)?

Reply:
May be it's worth it to move that stuff to it's own function

Reply:
Sure, it'd be a start at least.